composer/loggers/console_logger.py

```diff
@@ -84,21 +90,20 @@ def epoch_end(self, state: State, logger: Logger) -> None:
         unit = self.log_interval.unit
 
         if unit == TimeUnit.EPOCH and (cur_epoch % int(self.log_interval) == 0 or cur_epoch == 1):
-            log_dict = {**state.train_metric_values}
```

Comment:
Why are metrics only cleared on batch/epoch end? Should they be flushed on every event? or at least also `eval_end`?